account_statement_import_base/views/account_bank_statement_line.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" ?>
+<!--
+  Copyright 2022 Akretion France (http://www.akretion.com/)
+  @author: Alexis de Lattre <alexis.delattre@akretion.com>
+  Licence LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0).
+-->
+<odoo>
+
+<record id="view_bank_statement_line_form" model="ir.ui.view">
+    <field name="model">account.bank.statement.line</field>
+    <field name="inherit_id" ref="account.view_bank_statement_line_form" />
+    <field name="arch" type="xml">
+        <field name="statement_id" position="attributes">
+            <attribute
+                    name="invisible"
+                >not context.get('statement_line_main_view')</attribute>
+        </field>
+        <xpath expr="//field[@name='company_id']/.." position="attributes">
+            <attribute name="col">2</attribute>
+        </xpath>
+        <field name="sequence" position="attributes">
+            <attribute name="invisible">1</attribute>
+        </field>
+        <field name="narration" position="attributes">
+            <attribute name="invisible">1</attribute>
+        </field>
+        <field name="transaction_type" position="after">
+            <field name="partner_bank_id" />
+        </field>
+        <sheet position="inside">
+            <notebook>
+                <page name="narration" string="Notes">
+                    <field name="narration" nolabel="1" />
+                </page>
+                <page name="technical" string="Technical Information">
+                    <group name="tech-fields">
+                        <field name="unique_import_id" />
+                        <field name="partner_name" />
+                        <field name="account_number" />
+                        <field name="is_reconciled" />
```

Comment:
This field is not something of this module, so you shouldn't add it IMO as it's not directly related.

Reply:
unique_import_id is defined in account_statement_import_base.
partner_name, account_number and is_reconciled are defined in the "account" module but the account module doesn't display it in the form view. As those fields are very useful for technical audit of bank statement import (along with the field raw_data), I decided to create a notebook on the view, where the second tab is called "technical info" and it displays those fields. That way, the form view is not over-charged with technical fields which are only useful for technical audit in case of problem.

Reply:
But this is going a bit out of the scope. You see it as "I improve the usability a lot", meanwhile others like me see it as "this is breaking compatibility + not related with the change we are handling".

Reply:
Please show me exactly where you think I "break compatibility". I didn't move any field, I set the initial field as invisible and display it elsewhere AFTER the initial position. So, if you added a field via an inherit written <field name="narration" position="after">xxx</field>, then your added field would still be at the same place (after the field which is now invisible).

 I don't see how adding the 3 fields partner_name, account_number and is_reconciled would break compatibility.

Reply:
There are a lot of ways to break compatibility with this kind of changes. Examples:

- A form can only contains one notebook, and another module adding other notebook will have unpredictable behavior.
- Other modules can hook into any of the changed fields, so the fields won't be placed where wanted. This is not exactly breaking, but also not desired.
- Hiding fields will get mad someone that is expecting and using such fields for other purpose that you haven't predicted.
- In general, any change over the base visually speaking may be limited to the minimum.